client/jetpack-connect/remote-credentials.js

```diff
@@ -281,15 +285,28 @@ export class OrgCredentialsForm extends Component {
 	}
 
 	render() {
+		const { installError } = this.props;
+		const isFormInNotice = includes( [ LOGIN_FAILURE ], this.getError( installError ) );
```

Comment:
This can be simplified, of course. I left it for generality as the set of possible notice types that keep the form in  the view can be expanded. Since YAGNI, maybe we should change it to `===`?